### PR TITLE
fix(mail): Remove the "resend blackout period".

### PR DESCRIPTION
**Pull request by rfk**
Rate-limiting of outgoing mail is now handled by customs-server, we don't need special-case logic for it here.

Fixes https://github.com/mozilla/fxa-auth-mailer/issues/160

### CODE REVIEW

**Comment by rfk** [comment]
@dannycoates would you mind sanity-checking this for me please?

File: lib/routes/account.js [hunk 1]
```diff
@@ -972,10 +972,6 @@ module.exports = function (
         log.begin('Account.RecoveryEmailResend', request)
         var sessionToken = request.auth.credentials
         var service = request.payload.service || request.query.service
-        if (sessionToken.emailVerified ||
```
Comment (dannycoates) [inline lib/routes/account.js]: So we also want to skip short-circuiting if the email has been already been verified? Seems fine, I just want to confirm that its intentional.

Comment (rfk) [inline lib/routes/account.js]: Huh, no, that's not what I intended to do, thanks for catching it :-)

Comment (rfk) [inline lib/routes/account.js]: Although now that you mention it, I wonder if just sending the email again even if verified, is more likely to match user expectations...

**Comment by rfk** [comment]
@dannycoates thanks, updated to restore the "are you already verified?" check.

Also, /cc @vbudhram as this might merge conflict with the sign-in confirmation work

**Comment by dannycoates** [comment]
> Imma just take @dannycoates's comment as r+

whoops, I forgot to mention the r+ alongside my question, sorry

**Pull request merged**